Procfile

```diff
@@ -0,0 +1 @@
+web: bundle exec puma -C config/puma.rb
```

Comment:
While I like the idea of saving money by running everything in one dyno, shouldn't we still define the correct jobs in the Procfile so people can scale the dynos dynamically from the Heroku dashboard/CLI?